images/base/Dockerfile

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# runc (v1.0.0-rc93-pre)
+FROM golang:1.15-alpine AS runc
```

Comment:
if we're going to build with cgo shouldn't we build with matching libc?

Reply:
It is built as a static binary